src/components/singleReview.jsx

```diff
@@ -2,15 +2,21 @@ import { sliceDate } from "../utils/sliceDate";
 import { useParams } from "react-router-dom";
 import { useEffect, useState } from "react";
 import axios from "axios";
-
+import IncVotesBtn from "./incvotesbtn";
+import DecVotesBtn from "./decvotesbtn";
+import Loading from './loading'
 export default function SingleReview(){
     const {reviewId} = useParams()
     const [singleReview,setSingleReview] = useState({})
     const [user, setUsers] = useState({})
+    const [isLoading, setIsLoading] = useState(true)
+    const [disableInc, setDisableInc] = useState(false)
```

Comment:
The way that you are handling this means if I vote up once I can only go back one, i.e I give +1 vote I can only go to 0, If I clicked plus instead of minus I could never get to the stage that I wanted to in the first place :)